processing/src/main/java/org/apache/carbondata/processing/loading/events/LoadEvents.java

```diff
@@ -51,12 +51,6 @@ public LoadTablePreExecutionEvent(CarbonTableIdentifier carbonTableIdentifier,
       this.isOverWriteTable = isOverWriteTable;
     }
```

Comment:
i think `LoadTablePostExecutionEvent` also not used now, please check and remove

Reply:
Have checked and found a listener(MVLoadPostEventListener) for it.